src/reanimated2/platform-specific/webAnimationsData/Bounce.web.ts

```diff
@@ -0,0 +1,166 @@
+import { parseAnimationObjectToKeyframe } from '../animationParser';
+import type { AnimationData } from '../animationParser';
+
+const DEFAULT_BOUNCE_TIME = 0.6;
+
+export const BounceInData: Record<string, AnimationData> = {
```

Comment:
Is declaring it as a particular Record here necessary? Why won't we just do
```suggestion
export const BounceInData = {
```

Reply:
Done in [7f8735d](https://github.com/software-mansion/react-native-reanimated/pull/4805/commits/7f8735deb90c38024a96cbe8a49260758ee8097f)